stairway/src/test/java/bio/terra/stairway/impl/FilterTest.java

```diff
@@ -214,16 +214,16 @@ public void filterForm1PageTokenTest() throws Exception {
   @Test
   public void filterForm1OrderTest() throws Exception {
     String expect =
-            "SELECT F.flightid, F.stairway_id, F.submit_time, F.completed_time,"
-                    + " F.output_parameters, F.status, F.serialized_exception"
-                    + " FROM flight F"
-                    + " WHERE F.submit_time > :pagetoken"
-                    + " ORDER BY submit_time DESC LIMIT :limit";
+        "SELECT F.flightid, F.stairway_id, F.submit_time, F.completed_time,"
+            + " F.output_parameters, F.status, F.serialized_exception, F.class_name"
+            + " FROM flight F"
+            + " WHERE F.submit_time > :pagetoken"
+            + " ORDER BY submit_time DESC LIMIT :limit";
```

Comment:
IMO these kind of string comparison tests aren't very useful and the code is better served by tests that run the SQL. That being said, I would at least create strings for the common substrings to avoid needless code duplication.

Reply:
The main complexity in the filtering code is generating the right SQL. This kind of test makes it much more direct to ensure that is working, rather than debugging SQL errors inside the larger system.